rfcs/20200624-pluggable-device-for-tensorflow/sample/third_party/eigen.BUILD

```diff
@@ -0,0 +1,79 @@
+# Description:
+#   Eigen is a C++ template library for linear algebra: vectors,
```

Comment:
Since the whole `third_party/` folder is quite big, can we just download that folder from TensorFlow and keep only what we need? For example, we could add this to `build.sh` before the `bazel build` command. 
```
COMMIT=2eea5090c05a1cb99e631aecefd107597afd227b
wget https://github.com/tensorflow/tensorflow/archive/${COMMIT}.tar.gz -O tensorflow.tar.gz
tar -xvzf tensorflow.tar.gz tensorflow-${COMMIT}/third_party/ 
mv tensorflow-${COMMIT}/third_party/eigen3 third_party/
 ...  # Move other files.
rm -rf tensorflow-${COMMIT} tensorflow.tar.gz
```
This will make the demo code more compact.

Reply:
The demo only depends on eigen_archive and com_google_absl in third-party currently,  the version is the same as proper, do we need to download the whole tensorflow package(seems too big).